guides/plugins/apps/shipping-methods.md

```diff
@@ -0,0 +1,276 @@
+---
+nav:
+  title: Shipping methods
+  position: 25
+
+---
+
+# Shipping methods
+
+Starting with version REPLACE_WITH_NEXT_RELEASE_VERSION as **experimental feature**. Shopware has introduced experimental functionality for adding shipping methods via the App Manifest to a shop. **The entire functionality and API are subject to change during the development process.**
+
+## Prerequisites
+
+You should be familiar with the concept of Apps, their registration flow as well as signing and verifying requests and responses between Shopware and the App backend server.
+
+<PageRef page="app-base-guide" />
+
+Your app server must be also accessible for the Shopware server.
+
+## Manifest configuration
+
+### Basic configuration
+The following example represents the most minimal configuration for a shipping method.
+
+**Important!**
+
+Ensure that the `<identifier>` of your shipping method remains unchanged, as Shopware will deactivate or delete shipping methods that do no longer appear in the manifest during app updates.
+```xml
+// manifest.xml
+
+<?xml version="1.0" encoding="UTF-8" ?>
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-2.0.xsd">
+    <meta>
+        <!-- Make sure that the name of your app does not change anymore, otherwise there will be duplicates of your shipping methods -->
+        <name>NameOfYourShippingMethodApp</name>
+        <!-- ... -->
+    </meta>
+
+    <shipping-methods>
+
+        <shipping-method>
+            <!-- The identifier should not change after first release -->
+            <identifier>NameOfYourFirstShippingMethod</identifier>
+            <name>First shipping method</name>
+
+            <delivery-time>
+                <!-- Requires a new generated UUID for your new delivery time -->
```

Comment:
it is not required, isn't it? if not provided, we will create one?

Reply:
<xs:element type="delivery-time" name="delivery-time" minOccurs="1" maxOccurs="1"/>

currently it is required

Reply:
Currently testing it and left it out. The app then gets ignored. Do you know if that is logged somewhere? Not sure if that behavior is only like that for this field. It would be interesting in a debugging context.


![2023-10-18_14-17_1](https://github.com/shopware/docs/assets/8600299/6a4722dc-f1a0-477d-af1d-277b105924e9)
![2023-10-18_14-17](https://github.com/shopware/docs/assets/8600299/57d9e215-0a30-48e4-b9b3-ca652d31c153)

Reply:
I have tested it. At the moment it seems that there is nothing that is logged. I'll create a ticket that a defective manifest is displayed